files/.browserslistrc

```diff
@@ -0,0 +1,7 @@
+Chrome >= 60
+Firefox >= 60
+Edge >= 12
+Explorer >= 11
```

Comment:
I think we've effectively dropped IE support moving forward, so i think its safe to remove it here.